### ISSUE TITLE
Change Slider position

### ISSUE THREAD

**Issue by NikunjKhangwal** [butzist/ActivityLauncher#255]
Currently the slider is a bit on the left and it overlapas with text underneath.
![Screenshot_2024-03-30-21-35-58-075_de szalkowski activitylauncher](https://github.com/butzist/ActivityLauncher/assets/99175083/e09e1024-da6e-4f0f-9a5a-88717f7b85c4)

Move it to the right side a bit either slightly or completely like:-
![IMG_20240330_214245](https://github.com/butzist/ActivityLauncher/assets/99175083/39bc2a83-8bd3-435e-bb30-85c24d3662ad)

**Comment by butzist** [comment]
You mean fast scroll bar - should be easy to add some padding to the end.

**Comment by drogga** [comment]
It's probably better like it is now on/for phones with round (edge) screens, the trend moved to flat ones (again), but some brands keeps releasing "edge" ones, I have SGS8+, but the screen is out, so I'm not exactly sure, just a speculation.

**Comment by NikunjKhangwal** [comment]
Well flat screens are still significantly more than edge screens in numbers.

**Comment by drogga** [comment]
Can you please make the screenshots expandable, under details & summary ?, but use the Preview before editing the OP, to avoid unnecessary edits and potential mess ;)

**Comment by NikunjKhangwal** [comment]
Do you mean splitting this long screenshot?

**Comment by butzist** [comment]
For me the screenshots work and it's already clear what needs to be done. I.e. nevermind.

**Comment by drogga** [comment]
<details><summary>I actually meant</summary>like this, but NVM. There's no problem with them, the idea is to not load/see them every time someone loads the ticket.</details>

**Comment by butzist** [comment]
should be fixed in master - please confirm

**Comment by butzist** [comment]
reopen is still broken

**Comment by NikunjKhangwal** [comment]
Slider is still the same

**Comment by butzist** [comment]
I fixed the text margins on the activity list to not overlap with the scroll bar - want that the goal?

**Comment by butzist** [comment]
I can try to remove the extra margin on the right of the scroll bar in a follow-up release

**Comment by NikunjKhangwal** [comment]
Yeah do that. Removing overlapping works but the bar itself just looks weird in current position

**Comment by drewauff** [comment]
or add an option to remove it entirely

**Comment by butzist** [comment]
removed it as it was also causing other crashes. if someone needs it, let me know and/or create a PR that fixes it.
